.travis/test-part.sh

```diff
@@ -6,8 +6,8 @@ set -euxo pipefail
 readonly test_part=${TEST_PART:-}
 
 case "$test_part" in
-Sol6)
-    npx buidler test --no-compile --config buidlerConfigSol6.js
+Test)
```

Comment:
in test file can maybe leave both options
so later we can configure as we want for yml file

Reply:
actually seems more clear to call it All

and leave sol6 as well.